contracts/IdentityStaking.sol

```diff
@@ -123,20 +123,23 @@ contract IdentityStaking is
   /// @param staker The staker's address
   /// @param amount The additional amount added for this particular transaction
   /// @param unlockTime Unlock time for the full self-stake amount for this staker
+  /// @param lockDuration The duration of the lock
   /// @dev `amount` could be `0` for an extension
-  event SelfStake(address indexed staker, uint88 amount, uint64 unlockTime);
+  event SelfStake(address indexed staker, uint88 amount, uint64 unlockTime, uint64 lockDuration);
 
   /// @notice Emitted when a community stake is added/increased/extended
   /// @param staker The staker's address
   /// @param stakee The stakee's address
   /// @param amount The additional amount added for this particular transaction
   /// @param unlockTime Unlock time for the full community stake amount for this staker on this stakee
+  /// @param lockDuration The duration of the lock
   /// @dev `amount` could be `0` for an extension
   event CommunityStake(
     address indexed staker,
     address indexed stakee,
     uint88 amount,
-    uint64 unlockTime
```

Comment:
Do we need the `unlockTime` if we add the `lockDuration`?
The block number will be recorded anyways, can we just use that to determine the `lockTime` from the provider?

Just trying to avoid storing redundant data.